pages/overview/authentication.mdx

```diff
@@ -3,17 +3,24 @@ import { Link } from "@/components/Link";
 import { RichTabs } from "@/components/RichTabs";
 import { Code } from "@/components/Code";
 import { Screenshot } from "@/components/Screenshot";
-import { GitDiff, ChatText, Password } from "@phosphor-icons/react";
+import { GitDiff, UserFocus, Password } from "@phosphor-icons/react";
 import { ListDisclosure } from "@/components/ListDisclosure";
 import manifest from "@/data/manifest.json";
 
 # Authentication
 
-Before start setting up Auth.js, you need to decide how you're gonna authenticate users in your application. Auth.js supports <u>several authentication paradigms</u>:
+<Callout>
+  Auth.js discourages the use of passwords due to the inherent security risks of
+  the username-password model. Passwords are still supported through the
+  [external auth]((#setup-with-external-auth)) but we strongly encourage to
+  setup a combination of **OAuth + Passwordless** for authentication.
+</Callout>
```

Comment:
This one we actually need to remove, @balazsorban44 suggested is a bit of a polemical warning, fine to keep it in the **Credentials** tab 👍🏽